files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -406,6 +406,11 @@ audit-secureblue:
         "ia32_emulation=0"
     )
 
+# Install negativo repo
+install-negativo:
+    sudo curl -o /etc/yum.repos.d/negativo.repo https://negativo17.org/repos/fedora-multimedia.repo
```

Comment:
instead of calling sudo for each one, you can just do this:

```
#!/usr/bin/pkexec /usr/bin/bash
```

check some of the other commands above